tidb-cloud/release-notes-2022.md

```diff
@@ -8,6 +8,20 @@ aliases: ['/tidbcloud/beta/supported-tidb-versions','/tidbcloud/release-notes']
 
 This page lists the release notes of [TiDB Cloud](https://www.pingcap.com/tidb-cloud/) in 2022.
 
+## November 22, 2022
+
+**General changes**
+
+* Support directly migrating data from Amazon Aurora MySQL, Amazon Relational Database Service (RDS) MySQL, or self-built MySQL-compatible databases to TiDB Cloud online.
+
+    Previously, you needed to stop your business and import data offline, or use third-party tools to migrate data to TiDB Cloud, which was complicated. Now, with the **Data Migration** feature, you only need to perform operations on your TiDB Cloud console and securely migrate your data to TiDB Cloud with minimal downtime.
```

Comment:
```suggestion
    Previously, you needed to stop your cluster and import data offline, or use third-party tools to migrate data to TiDB Cloud, which was complicated. Now, with the **Data Migration** feature, you only need to perform operations on your TiDB Cloud console and securely migrate your data to TiDB Cloud with minimal downtime.
```

Reply:
the upstream business might be just on MySQL databases instead of a MySQL cluster